packages/gradle-plugin/react-native-gradle-plugin/src/test/kotlin/com/facebook/react/tasks/BundleHermesCTaskTest.kt

```diff
@@ -339,14 +332,14 @@ class BundleHermesCTaskTest {
 
     val hermesCommand = task.getHermescCommand(customHermesc, bytecodeFile, bundleFile)
 
-    assertEquals(customHermesc, hermesCommand[0])
-    assertEquals("-emit-binary", hermesCommand[1])
-    assertEquals("-max-diagnostic-width=80", hermesCommand[2])
-    assertEquals("-out", hermesCommand[3])
-    assertEquals(bytecodeFile.absolutePath, hermesCommand[4])
-    assertEquals(bundleFile.absolutePath, hermesCommand[5])
-    assertEquals("my-custom-hermes-flag", hermesCommand[6])
-    assertEquals(7, hermesCommand.size)
+    assertThat(hermesCommand[0]).containsExactly(customHermesc)
```

Comment:
@omer-ayhan Here with `containsExactly` I meant to assert all the parameters.

Like:

```
assertThat(hermesCommand).containsExactly(
  customHermesc,
  "-emit-binary",
  "-max-diagnostic-width=80"
...
)
```

Reply:
sorry my bad I'll do it as soon as I can @cortinico